server/src/migrations/1741731070431-SidecarInAssetFiles.ts

```diff
@@ -0,0 +1,29 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class Test1741731070431 implements MigrationInterface {
+  name = 'Test1741731070431';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`ALTER TABLE "assets" DROP COLUMN "sidecarPath"`);
```

Comment:
Should this move the existing sidecar paths over to asset_files rather than just dropping them?

Reply:
I'll think about this later